stdlib/io/src/main/java/org/ballerinalang/stdlib/io/nativeimpl/CharacterChannelUtils.java

```diff
@@ -146,4 +168,16 @@ public static Object writeXml(ObjectValue characterChannelObj, XMLValue content)
         }
         return null;
     }
+
+    public static Object writeProperties(ObjectValue characterChannelObj, BMap<BString, BString> propertyMap) {
+        try {
+            CharacterChannel characterChannel = (CharacterChannel) characterChannelObj
+                    .getNativeData(CHARACTER_CHANNEL_NAME);
+            String content = PropertyUtils.getWritablePropertyContent(propertyMap);
+            IOUtils.writeFull(characterChannel, content);
```

Comment:
Any reason to write properties using custom content rather than using Properties.store()

Reply:
Fixed in https://github.com/ballerina-platform/ballerina-lang/pull/24978/commits/61d9daade20c881f5fda49735f8a89516b5a3057. Both are quite same, here I changed it to store() since we need to add comments if needed, as specified in the Java store() function.